### PR TITLE
GiniPayBusiness -> GiniHealth

### PR DESCRIPTION
**Pull request by zladzeyka**
Documentation update:
- renamed Gini Pay Business SDK for iOS -> Gini Health SDK for iOS;
- example business app -> example health app;
- pod "GiniPayBusiness" - > pod "GiniHealth";
- renamed "GiniPayBusinessX" classes mentions in Integration guide -> "GiniHealth": GiniHealthConfiguration, GiniHealthError, GiniHealth;
- renamed `ginipay-business://payment-requester`.->`gini-health://payment-requester`;
- updated logo;
- updated license and readme (PIA-1830)
Example Swift:
- updated import GiniPayBusiness -> import GiniHealth;
- pod updated; (PIA-1830)
- changed related to showing an error when none of the picked files could be opened (PIA-1477)

### CODE REVIEW

File: Documentation/source/Customization guide.md [hunk 1]
```diff
@@ -24,26 +24,26 @@ Some background and text colors use the `GiniColor` type with which you can set
 </br>
 
 ##### 1. Background color
-- Background color &#8594; `GiniPayBusinessConfiguration.paymentScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
+- Background color &#8594; `GiniHealthConfiguration.paymentScreenBackgroundColor` using `GiniColor` with dark mode and light mode colors
```
Comment (a-szotyori) [inline Documentation/source/Customization guide.md:27]: I thought we agreed that the source code won't be renamed. This includes code snippets in the docs. We'll do the source code renaming and update the snippets in the docs at a later date.

I'm fine with it now that you already did the work 🙂. The Health SDK docs are anyhow just for show until we rename the source code and make the first public release.

File: Documentation/source/Integration.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 Integration
 =============================
 
-Gini Pay provides an information extraction system for analyzing business invoices and transfers them to the iOS banking app, where the payment process will be completed.
+Gini Health provides an information extraction system for analyzing business invoices and transfers them to the iOS banking app, where the payment process will be completed.
```
Comment (a-szotyori) [inline Documentation/source/Integration.md]:
```suggestion
The Gini Health SDK for iOS provides all the UI and functionality needed to use the Gini Pay API in your app to extract payment and health information from invoices. The payment information can be reviewed and then the invoice can be paid using any available payment provider app (e.g., banking app).
```

File: Documentation/source/Integration.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
-The Gini Pay Business SDK for iOS provides functionality to upload the multipage documents with mobile phones, accurate line item extraction enables the user to to pay the invoice with prefferable payment provider. 
+The Gini Health SDK for iOS provides functionality to upload the multipage documents with mobile phones, accurate line item extraction enables the user to to pay the invoice with prefferable payment provider.
```
Comment (a-szotyori) [inline Documentation/source/Integration.md]:
```suggestion
The Gini Pay API provides an information extraction service for analyzing health invoices. Specifically, it extracts information such as the document sender or the payment relevant information (amount to pay, IBAN, etc.). In addition it also provides a secure channel for sharing payment related information between clients. 
```

**Review by a-szotyori — COMMENTED** [review]
Just two small changes regarding the introduction.

File: Documentation/source/Customization guide.md (hunk 1 above)
Comment (zladzeyka) [reply Documentation/source/Customization guide.md:27]: overdone.

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**